dataobj/schedule.cc

```diff
@@ -683,6 +732,47 @@ void schedule_t::set_spacing_for_all(uint16 v) {
 	}
 }
 
+
+// check if the next line is set.
+// if the next line is wrong, unset it.
```

Comment:
このコメントはheader fileの方に書くようにしてください。
```suggestion
// Returns true if the given line can be the next line of this schedule.
```